indico/core/emails.py

```diff
@@ -108,9 +108,10 @@ def do_send_email(email, log_entry=None, _from_task=False):
                        the celery task responsible for sending emails.
     """
     with get_connection() as conn:
-        msg = EmailMessage(subject=email['subject'], body=email['body'], from_email=email['from'],
-                           to=email['to'], cc=email['cc'], bcc=email['bcc'], reply_to=email['reply_to'],
-                           attachments=email['attachments'], connection=conn)
+        msg = EmailMultiAlternatives(subject=email['subject'], body=email['body'], from_email=email['from'],
+                                     to=email['to'], cc=email['cc'], bcc=email['bcc'], reply_to=email['reply_to'],
+                                     attachments=email['attachments'], connection=conn,
+                                     alternatives=email['alternatives'])
```

Comment:
I think it's safer to use `.get()` here since an email that got enqueued but not sent yet right before updating Indico would not have this key and thus fail.